nylas/client/restful_models.py

```diff
@@ -83,6 +93,12 @@ class Message(NylasAPIObject):
              "account_id", "object", "snippet", "starred", "subject",
              "thread_id", "to", "unread", "starred", "_folder", "_labels",
              "headers"]
+    datetime_attrs = {
+        "received_at": "date",
+        # The following are used for filtering:
```

Comment:
We could, but they're treated exactly the same as the other datetime attributes. Do you think it's worth the duplication? I figured that a comment would make it less confusing, but if you think we need to go further and have a separate field, I can do that.